Dockerfile

```diff
@@ -8,28 +8,29 @@ RUN  ls && npm run build
 
 FROM golang:1.17 as builder
 WORKDIR /app
-COPY ./ ./
-
+COPY go.mod go.mod
+COPY go.sum go.sum
 ARG VERSION
-COPY go.mod /app/go.mod
-COPY go.sum /app/go.sum
 RUN go mod download
 COPY --from=node /app/build /app/ui/build
-WORKDIR /app
-RUN go version
+COPY ./ ./
 RUN make build
+
 FROM ubuntu:bionic
 WORKDIR /app
 
 # install CA certificates
 RUN apt-get update && \
-  apt-get install -y ca-certificates && \
+  apt-get install -y ca-certificates curl && \
   rm -Rf /var/lib/apt/lists/*  && \
   rm -Rf /usr/share/doc && rm -Rf /usr/share/man  && \
   apt-get clean
 
 COPY --from=builder /app/.bin/incident-commander /app
 
 RUN /app/incident-commander go-offline
+RUN curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s
```

Comment:
Do we need this if compose is using the air image? Shouldn't it be a in the make file as a dependency when running outside docker

Reply:
Yeah, this is not needed. Removed it from dockerfile